src/envoy/http/jwt_auth/jwt_authenticator_test.cc

```diff
@@ -107,6 +107,13 @@ const char kExampleConfig[] = R"(
 }
 )";
 
+// An example JSON config with allow_missing_or_failed option enabled
+const char kExampleConfigWithAllowMissingOrFailed[] = R"(
+{
```

Comment:
You test config have no jwt spec, what use cases can this cover?

Reply:
Done: added comments to explain the test case (when jwt is missing, status in the callback should be OK).